static/js/whyis_vue/components/table-view/components/columnaddition.vue

```diff
@@ -0,0 +1,22 @@
+<template>
```

Comment:
File name should be the same as the component name, in this case ColumnAddition.vue.

Reply:
Renamed